mina-p2p-messages/src/v2/manual.rs

```diff
@@ -713,6 +714,26 @@ pub type NonZeroCurvePoint = Base58CheckOfBinProt<
     { crate::b58version::NON_ZERO_CURVE_POINT_COMPRESSED },
 >;
 
+#[derive(Clone, Debug, PartialEq, Serialize, Deserialize, BinProtRead, BinProtWrite)]
+pub enum ArchiveTransitionFronntierDiff {
+    BreadcrumbAdded {
+        block: (MinaBlockBlockStableV2, (Option<StateBodyHash>, StateHash)),
+        accounts_accessed: List<(crate::number::UInt64, MinaBaseAccountBinableArgStableV2)>,
+        accounts_created: List<(MinaBaseAccountIdStableV2, CurrencyFeeStableV1)>,
+        tokens_used: List<(MinaBaseTokenIdStableV2, Option<MinaBaseAccountIdStableV2>)>,
+        sender_receipt_chains_from_parent_ledger:
+            List<(MinaBaseAccountIdStableV2, MinaBaseReceiptChainHashStableV1)>,
+    },
+    // TODO(adonagy): I think this is legacy stuff, doublecheck
```

Comment:
were you able to figure out if this is used at all? (I couldn't find any use myself, but it wasn't a super thorough search)

Reply:
Not used. It looks like a remnant of some legacy code. Those 2 variants are not used or built anywhere.